include/stdlib.h

```diff
@@ -12,105 +12,785 @@ extern "C" {
 
 typedef struct
```

Comment:
Can we add comments for the three structs? integer division type, long integer, long long.